compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -27,6 +27,46 @@ using Granularity = luci::QuantizationGranularity;
 
 namespace
 {
+// helper function to create dummy float const node at the graph
+template<Type T>
+luci::CircleConst *createDummyConst(loco::Graph *g, luci::test::ShapeU32 shape)
+{
+  auto node = g->nodes()->create<luci::CircleConst>();
+  {
+    node->dtype(T);
+    node->shape(shape);
+    node->size<T>(luci::test::num_elements(shape));
+
+    for (int32_t i = 0; i < luci::test::num_elements(shape); i++)
+    {
+      // DESIGN NOTE
+      //
+      // Filling with any random numbers are fine
+      // Q. Should it include minus numbers?
```

Comment:
I think the current values are ok for now. If we need more complicated tests which require to set specific values, then let's add new API at that time.